apps/core/components/rangeDatePicker/RenderMonth.js

```diff
@@ -0,0 +1,75 @@
+// @flow
+
+import * as React from 'react';
+import { View } from 'react-native';
+import { StyleSheet } from '@kiwicom/universal-components';
+import { Text } from '@kiwicom/margarita-components';
+import { defaultTokens } from '@kiwicom/orbit-design-tokens';
+import { format } from 'date-fns';
+import { MONTH_NAME_FORMAT } from '@kiwicom/margarita-config';
+import memoize from 'memoize-one';
+import { equals } from 'ramda';
+
+import RenderWeek from './RenderWeek';
+import { type MonthDate, getMonthMatrix } from './libs';
+
+type Props = {|
+  +monthDate: MonthDate,
+  +onDayPress: Date => void,
+  +selectedDate: Date,
+|};
+
+export default class RenderMonth extends React.Component<Props> {
+  shouldComponentUpdate(nextProps: Props) {
+    const checkDate = props =>
+      props.selectedDate &&
+      props.selectedDate.getMonth() === props.monthDate.month &&
+      props.selectedDate.getFullYear() === props.monthDate.year;
+    return checkDate(nextProps) || checkDate(this.props);
```

Comment:
what about just using `import isEqual from 'react-fast-compare';`

Reply:
How would you use it?  `props.selectedDate` and `props.monthDate` is not equal.

Reply:
You are right, forget that comment 🙈